product/opni/router.js

```diff
@@ -25,7 +25,7 @@ export const NAVIGATION = {
       name:      'clusters',
       path:      '/clusters',
       labelKey:  'opni.nav.clusters',
-      icon:      'globe',
```

Comment:
For now I think this is fine but when we integrate into rancher these icons are currently used to signify a namespaces vs global resource.